nulib/next-gen-front-end-react
# Switch from honeybadger-js to @honeybadger-io/react

## Summary 

Replace legacy `honeybadger-js` module with `@honeybadger-io/react`

## Specific Changes in this PR
- Update dependencies
- Add `HONEYBADGER_REVISION` variable
- Insert the error boundary

## Steps to Test

@adamjarling Honeybadger error reporting still wasn't working after the last update, so I updated it to the newer, React-specific Honeybadger package and configuration as described [in the Honeybadger docs](https://docs.honeybadger.io/lib/javascript/integration/react/#installation). I'm not sure how best to test it in development, but I thought you should look it over to make sure it looks right.
